libs/javascript.json

```diff
@@ -64,6 +64,7 @@
   {"id":"js.validate","imports":["validate"],"name":"validate","repo":"jquery-validation/jquery-validation","tags":["jquery-validation"],"tech":["javascript-web"],"status":"model-ready"},
   {"id":"js.vue","imports":["vue"],"name":"vue","repo":"vuejs/vue","tags":["ui-framework"],"tech":["javascript-web"],"status":"model-ready"},
   {"id":"js.vuex","imports":["vuex"],"name":"vuex","repo":"vuejs/vuex","tags":["state-management"],"tech":["javascript-web"],"status":"model-ready"},
+  {"id":"js.ml5","imports":["ml5"],"name":"ml5","repo":"ml5js/ml5-library","tags":["framework"],"tech":["ml"],"status":"awaiting-model"},
```

Comment:
could you replace tags with 'toolkit' instead of 'framework'? We like to call it a 'framework' if it likes to define your app structure, and generally is not usable as a drop-in.

Reply:
Yes, that makes totally sense.